sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -194,7 +201,7 @@ await using (var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsum
                         var batchNumber = 1;
                         var batchSize = (eventBatch.Length / 3);
 
-                        while ((receivedEvents.Count < eventBatch.Length) && (++index < eventBatch.Length + 5))
+                        while ((receivedEvents.Count < eventBatch.Length) && (++index < eventBatch.Length + 10))
```

Comment:
nit:  Missed substituting the constant here.

Reply:
Done.